entgql/template/where_input.tmpl

```diff
@@ -77,42 +77,51 @@ in the LICENSE file in the root directory of this source tree.
         i.Predicates = append(i.Predicates, predicates...)
     }
 
+    {{ $err := printf "ErrEmpty%s" $input }}
+
     // Filter applies the {{ $input }} filter on the {{ $n.QueryName }} builder.
     func (i *{{ $input }}) Filter(q *{{ $n.QueryName }}) (*{{ $n.QueryName }}, error) {
         if i == nil {
             return q, nil
         }
         p, err := i.P()
         if err != nil {
+            {{- /* Skip errors only for top-level empty predicates. The rest are invalid. */}}
+            if err == {{ $err }} {
+                err = nil
+            }
             return nil, err
```

Comment:
@a8m: we should return q, instead of nil.

Reply:
Right! Thanks